config.w32

```diff
@@ -20,9 +20,14 @@ if (PHP_MONGODB != "no") {
 
   AC_DEFINE("HAVE_MONGOC", 1);
 
-  if (!CHECK_LIB("libsasl.lib", "mongodb", PHP_MONGODB)) {
-	  WARNING("mongodb libsasl support not enabled, libs not found");
+  if (PHP_MONGODB_SASL != "no") {
```

Comment:
This is based on what we were doing for 1.x [here](https://github.com/mongodb/mongo-php-driver/blob/239a4324911094ff6c755afaf78aadc3981194b1/config.w32).